### DISCUSSION TITLE
Error laragon detected exceptions!

### DISCUSSION THREAD

**Discussion by IvanMGC** [leokhoa/laragon#590] [General]
![image](https://github.com/leokhoa/laragon/assets/83800226/4d760303-e784-48e3-b5b0-21071231f111)
Yesterday it worked wonderfully for me and today it didn't happen, it doesn't tell me what error it is, but it appears. Please help me resolve my case.

**Comment by leokhoa** [comment]
@IvanMGC Did you have any Windows update? Try rolling back.

**Comment by IvanMGC** [comment]
I started using Laragon again yesterday and updated Laragon with the same version of Windows 11 that I have.

**Reply by IvanMGC** [reply]
and it works great for me, today I wanted to use it again and I got that error, I don't know if anything has been installed.
